sips/sip-022/sip-022-emergency-pox-fix.md

```diff
@@ -48,6 +48,9 @@ Stacks 2.3.
 
 # Introduction
 
+**Note**: [SIP-023](./sips/sip-023/sip-023-emergency-fix-traits.md), which was accepted by the required CAB's on May 2nd, 2023 necessitates some changes to this SIP. The changes introduced in [SIP-023](./sips/sip-023/sip-023-emergency-fix-traits.md) are in response to a bug introduced by the first hard fork here, referred to later in this SIP as `Stacks 2.2`. To address the issue, an intermediary version `Stacks 2.3` was created which supercedes the second hard fork as defined here. 
```

Comment:
Maybe instead of just `**Note**`, we should make this stand out more, with a title and then have the whole paragraph in italics? Otherwise, sounds good to me.

### **_Addendum added after ratification_**

Reply:
Per SIP-000 (missed this when i had checked), new sections can be added as required. 
I agree with you that this applies here, so i've created a new `Addendum` section before `Introduction` and moved the text there with a brief sentence explaining why it exists.